src/layout.jl

```diff
@@ -192,7 +192,7 @@ Vector of Vector, Vector of node Vector for each shell.
 julia> g = graphfamous("karate")
```

Comment:
We should stop using graphfamous eventually, because it is taken out of this package. The karate graph can then be generated using `smallgraphs(:karate)`. Unfortunately this does not work yet, because we have to wait until a new version of LightGraphs is getting tagged.

Reply:
Ah, just realised that was not put in by your pr, so forget about that.